pkgs/development/python-modules/zephyr-python-api/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, requests
+, buildPythonPackage
+, fetchPypi
+, setuptools
```

```suggestion
, buildPythonPackage
, fetchPypi
, requests
, setuptools
```
Usually, we sort inputs alphabetically.

---

Done.